configuration/pih/htmlforms/hiv/section-hiv-plan_v3.0.xml

```diff
@@ -1260,26 +1260,12 @@
 
     </section>
 
-    <script>
-        jq(function() {
-            let providerBox = jq("#provider-name-non-coded input");
-            if(providerBox.val()) {
-                providerBox.attr("disabled", "disabled");
-            } else {
-                jq("#provider").hide();
-            }
-        });
-    </script>
-
-    <section id="provider" sectionTag="fieldset"
-             headerTag="legend" headerStyle="title"
-             headerCode="pihcore.provider">
-
+    <section sectionTag="fieldset" headerTag="legend" headerStyle="title" headerCode="pihcore.provider">
         <div class="section-container-color">
             <label>
                 <uimessage code="pihcore.provider"/>
             </label>
-            <obs id="provider-name-non-coded" conceptId="PIH:Provider name non-coded" disabled="disabled" size="50" />
+            <obs conceptId="PIH:Provider name non-coded" disabled="disabled" size="50" />
```

Comment:
I'd actually get rid of the disabled as well... users should be able to edit (as this will only be on old forms)